assignments/closure.js

```diff
@@ -3,20 +3,36 @@
 // Keep it simple! Remember a closure is just a function
 // that manipulates variables defined in the outer scope.
 // The outer scope can be a parent function, or the top level of the script.
+let globalVariable = 1;
 
+let closureFunction = function() {
+  return 5 + globalVariable;
+}
 
 /* STRETCH PROBLEMS, Do not attempt until you have completed all previous tasks for today's project files */
 
 
 // ==== Challenge 2: Implement a "counter maker" function ====
-const counterMaker = () => {
+const counterMaker = (limit) => {
+	let count = 0;
+	let counter = () => {
+		count >= limit ? count = 0 : count += 1;
+		return count;
+	}
```

Comment:
This logic also doesn't work. You might want to take a look at your conditional statement and how you are instantiating your variables